arduino/control-modules/i2c-reader.js

```diff
@@ -2,30 +2,33 @@ const eventBus = require('../../events/event-bus')
 const { I2C_DATA } = require('../sensor-keys')
 const { I2C_DATA_RECIEVED } = require('../../events/event-keys')
 
+const READ_DELAY = 1000
+
 function handleChannel (board, channelNumber) {
-  board.io.i2cRead(channelNumber, 27, function (arrayOfBytes) {
-    const length = arrayOfBytes.length
-    let message = ''
+  return setInterval(function () {
+    board.io.i2cReadOnce(channelNumber, 27, function (arrayOfBytes) {
+      let message = ''
+      let rawMessage = arrayOfBytes.filter(el => el !== 255)
 
-    console.log('arrayOfBytes', arrayOfBytes)
+      if (rawMessage.length) {
+        rawMessage.forEach(code => {
+          const char = String.fromCharCode(code)
+          if (code !== 255) {
```

Comment:
1. it looks to be double filtering, it was done on the line:11
>  let rawMessage = arrayOfBytes.filter(el => el !== 255)
1. it might be written shortly in the form - `message = String.fromCharCode(...arrayOfBytes.filter(el => el !== 255))`

Reply:
additionally, 
1. if we use strings then we can just terminate a message by a special character - new line symbol for example `array.slice(0, array.indexOf())`
2. if we use bytes only then we might try to use the max length of the message and actual length, in our case just 4 bytes. <type><length><data><data>